otlp/docker/.env

```diff
@@ -1,2 +1,3 @@
 OTELCOL_IMG=otel/opentelemetry-collector-contrib:latest
 OTELCOL_ARGS=
+COLLECTOR_OTLP_ENABLED=true
```

Comment:
to make it clearer that this is for jaeger, can we move this to the docker compose file? e.g.

```
  jaeger-all-in-one:
    image: jaegertracing/all-in-one:latest
    ports:
      - "16686:16686"
      - "14268"
      - "14250:14250"
    environment:
      - COLLECTOR_OTLP_ENABLED=true
```